### PR TITLE
Lidar post

### PR DESCRIPTION
**Pull request by rkelson**
add images and create a new post about the lidar acquisitions

### CODE REVIEW

**Comment by steveoh** [comment]
you forgot the `.md` file extension

File: _posts/2020-05-13-2020-utah-lidar-acquisition.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  email: rkelson@utah.gov
+date: 2020-05-13 12:10:00
+categories:
+- Featured
+tags:
+- lidar
+- 3dep
+---
+
+AGRC has awarded Aero-Graphics, Inc. and The Sanborn Map Company, Inc. to acquire [*20,958 square miles*](http://arcg.is/08W4bS "view project areas in ArcGIS Online") of aerial lidar elevation data. The lidar will be a combination of USGS Quality Level 1 (_QL1_), 8 points per square meter and Quality Level 2 (_QL2_), 2 points per square meter depending on the area. The 2019 QL2 project covering portions of Kane County, Utah was acquired last fall, is going through USGS Quality Assurance, and is scheduled to be completed by June 30, 2020. The 2020 acquisitions have begun and will continue throughout the year. These project areas will be completed in 2021 and project deliverables are scheduled to be completed by June 30, 2021.
```
Comment (steveoh) [inline _posts/2020-05-13-2020-utah-lidar-acquisition.md:14]: **nitpick**: if you want the links to be bold it's two `*`'s if you want it italic it's 1 `_`. Same for the link in the next paragraph.

File: _posts/2020-05-13-2020-utah-lidar-acquisition.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+The Federal Emergency Management Agency, the U.S. Forest Service, and the U.S. Geological Survey (USGS) National Map 3D Elevation Program (3DEP) are also acquiring [*13,681 square miles*](http://arcg.is/08W4bS "view project areas in ArcGIS Online") of aerial lidar in Utah this year. The lidar will mostly be QL2, with a few small aress as QL1. The acquisitions will begin in the fall of 2020 and project deliverables are scheduled to be completed by June 30, 2022.
```
Comment (steveoh) [inline _posts/2020-05-13-2020-utah-lidar-acquisition.md:16]: fix spelling 

```suggestion
The Federal Emergency Management Agency, the U.S. Forest Service, and the U.S. Geological Survey (USGS) National Map 3D Elevation Program (3DEP) are also acquiring [*13,681 square miles*](http://arcg.is/08W4bS "view project areas in ArcGIS Online") of aerial lidar in Utah this year. The lidar will mostly be QL2, with a few small areas as QL1. The acquisitions will begin in the fall of 2020 and project deliverables are scheduled to be completed by June 30, 2022.
```

**Comment by rkelson** [comment]
thanks for the review. Looks like a masterpiece to me

File: _posts/2020-05-13-2020-utah-lidar-acquisition.md (hunk 1 above)
Comment (rkelson) [reply _posts/2020-05-13-2020-utah-lidar-acquisition.md]: done

**Pull request merged**